src/client/pfs/pfs.proto

```diff
@@ -644,6 +648,8 @@ service API {
   rpc InspectFileV2(InspectFileRequest) returns (FileInfo) {}
   // WalkFileV2 walks over all the files under a directory, including children of children.
   rpc WalkFileV2(WalkFileRequest) returns (stream FileInfo) {}
+  // ClearCommit removes all data from the commit.
+  rpc ClearCommit(ClearCommitRequest) returns (google.protobuf.Empty) {}
```

Comment:
This should probably have the V2 suffix since it does not have a V1 implementation.